soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -327,6 +336,12 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
         CustomMessageManager::Instance->CreateMessage(
             Randomizer::hintMessageTableID, hintLocation.check, { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, hintLocation.hintText, hintLocation.hintText, hintLocation.hintText });
     }
+
+    CustomMessageManager::Instance->CreateMessage(Randomizer::hintMessageTableID, TEXT_WARP_RANDOM_REPLACED_TEXT,
+        { TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM,
+        "Warp to&{{location}}?\x1B&%gOK&No%w\x02",
+        "Warp to&{{location}}?\x1B&%gOK&No%w\x02", // TODO: German translation
+        "Warp to&{{location}}?\x1B&%gOK&No%w\x02" }); // TODO: French translation
```

Comment:
@PurpleHato If you could help with a translation here 😄 that would be nice.

This would either say something like `"Warp to Lake Hylia"`, or if hints are turned off, it will say `"Warp to a mysterious place"`

Reply:
Hmm I see..
I think that may be out of scope for this PR as that sounds like needing a preposition in all languages assigned to every hint area, and some of the entrances that are piped into this aren't based on the hints, an instead fallback to a hardcoded name that is only English.

What could work in the interim until we figure that out?

Reply:
I'm back home and had time to think about it, for now, let's just do
"Se téléporter: place name!
OK!
Non
"
for the OK! and Non is literally how they write it in french
![image](https://user-images.githubusercontent.com/47987542/205976411-872823a2-1245-4ce2-8b8e-3fb0f6f7ce53.png)